infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/resources/ghproxy-service.yaml

```diff
@@ -0,0 +1,31 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: ghproxy
+  namespace: default
+  name: ghproxy
+spec:
+  ports:
+    - name: main
+      port: 80
+      protocol: TCP
+      targetPort: 8888
+    - name: metrics
+      port: 9090
+  selector:
+    app: ghproxy
+  type: NodePort
```

Comment:
This should be `ClusterIP`, we don't want this exposed outside of the cluster

We may want metrics available outside of this cluster, I would create a separate service for that endpoint.  See how this is done for boskos or greenhouse for examples